packages/data-service-generator/src/utils/ast.ts

```diff
@@ -443,15 +517,19 @@ export function importContainedIdentifiers(
   const nameToId = Object.fromEntries(
     Array.from(idToModule.keys(), (identifier) => [identifier.name, identifier])
   );
+
   const containedIds = findContainedIdentifiers(node, idToModule.keys());
+
   const moduleToContainedIds = groupBy(containedIds, (id) => {
     const knownId = nameToId[id.name];
     const module = idToModule.get(knownId);
     return module;
   });
-  return Object.entries(moduleToContainedIds).map(([module, containedIds]) =>
-    importNames(containedIds, module)
+  const res = Object.entries(moduleToContainedIds).map(
+    ([module, containedIds]) => importNames(containedIds, module)
   );
+
+  return res;
```

Comment:
Is this change (getting `res` and then returning it instead of just returning the result as in the original code) needed/has any meaning? If not, let's revert this change for clarity.